_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: seo drupal drupal8
```

Comment:
Whoops, you did add seo, but you didn't add the tag completely, so it doesn't show up on the post.